compose.yaml

```diff
@@ -61,12 +61,6 @@ services:
       - AWS_SESSION_TOKEN
       - AWS_CREDENTIAL_EXPIRATION
 
-  sycamore_crawler_http_sort_one:
```

Comment:
We don't want to remove it, we want to leave it in but not run it by default.  If someone downloads the quickstart we want to have a way for them to import one file -- it's much faster that way (~100x) and it gets people started.

Reply:
This should be an option for the crawler. maybe a --sort-one and then the already used --sort-all

Reply:
In Jon's document, he mentions
> "Remove the automatic download and ingestion of a PDF. Include that PDF with sort-all parameter."
This is false proposition since this PDF is already included in the sort-all.

Reply:
Yes. I think Eric is suggesting we add a sort_one option to the crawler, what that will download the file that was previously downloaded automatically.

Reply:
Correct.